int32 seq 
int32 cmd 
--- 
string status

